fix path locations.  this is a gobo issue since the default puts rungetty in
/sbin and the mans in /usr/man/man8 which is right for just about every other
distro :)

--- rungetty-1.2/Makefile	1999-02-04 12:21:04.000000000 -0600
+++ rungetty-1.2/Makefile.new	2009-01-17 06:40:04.000000000 -0600
@@ -7,9 +7,9 @@
 
 install:	all
 		mkdir -p $(ROOT)/sbin
-		mkdir -p $(ROOT)/usr/man/man8
+		mkdir -p $(ROOT)/man/man8
 		install -s -m 0755 -o root -g root rungetty $(ROOT)/sbin/
-		install -m 0644 -o root -g root rungetty.8 $(ROOT)/usr/man/man8/
+		install -m 0644 -o root -g root rungetty.8 $(ROOT)/man/man8/
 
 rungetty:	rungetty.o
 
